### ISSUE TITLE
Unsupported JSON schema property type 'date' for property 'createdAt'

### ISSUE THREAD

**Issue by IceOnFire** [feathers-plus/generator-feathers-plus#246]
### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))

- [ ] Tell us what broke. The more detailed the better.
- [ ] If you can, please create a simple example that reproduces the issue and link to a gist, jsbin, repo, etc.

I'm having trouble using dates on my services. I think that the format 'date-time' is needed only for Sequelize, but is it normal that it brakes when I use MongoDB?

These are the steps to reproduce the problem:

1. `feathers-plus generate service`

```
? What is the name of the service? users
? What would you call one row in the users database? user
? What kind of service is it? MongoDB
? Place service code in which nested folder, e.g. `v1/blog`? (optional) 
? Which path should the service be registered on? /users
? Does the service require authentication? Yes
? Should this be served by GraphQL? Yes
```

2. Add a `createdAt: { format: 'date-time' }` to schema

3. `feathers-plus generate service`

### Expected behavior
Tell us what should happen

The service should be re-generated without errors

### Actual behavior
Tell us what happens instead

```
Error: Unsupported JSON schema property type 'date' for property 'createdAt'
    at JsonSchemaDefinition.compileProperty (/home/antony/.config/yarn/global/node_modules/sequelizer/lib/JsonSchemaDefinition.js:100:19)
    at JsonSchemaDefinition.compileAllProps (/home/antony/.config/yarn/global/node_modules/sequelizer/lib/JsonSchemaDefinition.js:138:14)
    at Function.fromJsonSchema (/home/antony/.config/yarn/global/node_modules/sequelizer/index.js:29:8)
    at serviceSpecsToSequelize (/home/antony/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/lib/service-specs-to-sequelize.js:37:31)
    at service (/home/antony/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/generators/writing/service.js:188:32)
    at generatorWriting (/home/antony/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/generators/writing/index.js:236:7)
    at ServiceGenerator.writing (/home/antony/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/generators/service/index.js:261:5)
    at Object.<anonymous> (/home/antony/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/node_modules/yeoman-generator/lib/index.js:424:27)
    at /home/antony/.config/yarn/global/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /home/antony/.config/yarn/global/node_modules/run-async/index.js:24:19
    at self.env.runLoop.add.completed (/home/antony/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/node_modules/yeoman-generator/lib/index.js:425:13)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
Emitted 'error' event at:
    at Immediate.setImmediate (/home/antony/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/node_modules/yeoman-generator/lib/index.js:433:22)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

```
    "@feathersjs/configuration": "^2.0.6",
    "@feathersjs/errors": "^3.3.6",
    "@feathersjs/express": "^1.3.1",
    "@feathersjs/feathers": "^3.3.1",
```

```
    "ts-node": "^7.0.1",
    "tslint": "^5.13.0",
    "typescript": "^3.3.3333"
```

**NodeJS version**:

v10.15.2

**Operating System**:

Ubuntu 18.10

**Browser Version**:

No browser required

**React Native Version**:

No React Native required

**Module Loader**:

No Module Loader required

**Comment by eddyystop** [comment]
@marshallswain could you comment on this?

**Comment by florianbepunkt** [comment]
I'm experiencing the same issue since I upgraded generator-feathers-plus today

**Comment by eddyystop** [comment]
PR https://github.com/feathers-plus/generator-feathers-plus/pull/236 published as v0.8.8 enhanced the JSON-schema to mongoDB model conversion.

Please use f+/cli v0.8.7 until we resolve this issue.

**Comment by MohammedFaragallah** [comment]
same here started with generator version 0.8.6

0.8.5 work

**Comment by eddyystop** [comment]
Published f+/cli v0.8.9 which uses generator v0.8.9
